### PR TITLE
Feature/create visit

### PR DESCRIPTION
**Pull request by alexsorindrago**
model of create customer visit
need review info page

### CODE REVIEW

File: src/app/features/create-visit/create-visit.component.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  styleUrls: ['./create-visit.component.scss']
+})
+export class CreateVisitComponent implements OnInit {
+  isLinear = false;
+  firstFormGroup: FormGroup;
+  secondFormGroup: FormGroup;
+  tyreDetailsFormGroup: FormGroup;
+
+
+  constructor(private _formBuilder: FormBuilder) { }
```
Comment (cosminbucur) [inline src/app/features/create-visit/create-visit.component.ts:16]: avoid using _ prefix for variables

File: src/app/features/create-visit/create-visit.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  ngOnInit() {
+    this.firstFormGroup = this._formBuilder.group({
```
Comment (cosminbucur) [inline src/app/features/create-visit/create-visit.component.ts:19]: this entire code block can be extracted into a method. though I think something is missing. is the save button working?

File: src/app/features/create-visit/create-visit.component.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
+<button mat-raised-button (click)="isLinear = !isLinear" id="toggle-linear">
+    {{!isLinear ? 'Enable linear mode' : 'Disable linear mode'}}
+  </button>
+  <mat-horizontal-stepper [linear]="isLinear" #stepper>
+    <mat-step [stepControl]="firstFormGroup">
+      <form [formGroup]="firstFormGroup">
+        <ng-template matStepLabel>Client details</ng-template>
+        <mat-form-field>
+          <mat-label>Last Name</mat-label>
+          <input matInput placeholder="ex: Eminescu" formControlName="firstCtrl" required>
```
[the remark this answers is not in the document]
Comment (alexsorindrago) [reply src/app/features/create-visit/create-visit.component.html]: fixed

File: src/app/features/create-visit/create-visit.component.ts (hunk 1 above)
Comment (alexsorindrago) [reply src/app/features/create-visit/create-visit.component.ts]: fixed

File: src/app/features/create-visit/create-visit.component.ts (hunk 2 above)
Comment (alexsorindrago) [reply src/app/features/create-visit/create-visit.component.ts]: help? :))

**Pull request merged**